SmartDeviceLink/SDLStreamingMediaLifecycleManager.m

```diff
@@ -784,7 +824,17 @@ - (void)sdl_sendVideoStartService {
 }
 
 
-#pragma mark Getters
+#pragma mark Setters / Getters
+
+- (void)setRootViewController:(UIViewController *)rootViewController {
+    if (self.focusableItemManager != nil) {
+        self.focusableItemManager.viewController = rootViewController;
+    }
+
+    if (self.carWindow != nil) {
+        self.carWindow.rootViewController = rootViewController;
+    }
```

Comment:
This only works for the first `rootViewController`. Setting a new `rootViewController` crashes the app because the frame of the new `rootViewController` is the same as the dimensions of the device screen instead of the SDL accessory screen size. The frame needs to be reset again every time a `rootViewController` is set.

```
self.rootViewController.view.frame = CGRectMake(0, 0, self.streamManager.screenSize.width, self.streamManager.screenSize.height);
self.rootViewController.view.bounds = self.rootViewController.view.frame;
```

Reply:
Should be fixed